File: Modulo-Fundamentos-Web/bloco_4/dia_1/Exercises/triangle.js
// Faça um programa que defina três constantes com os valores dos três ângulos internos de um triângulo.Retorne true se os ângulos representarem os ângulos de um triângulo e false, caso contrário.Se algum ângulo for inválido o programa deve retornar uma mensagem de erro.

const sideA = 40;
const sideB = 40;
const sideC = 100;
let triangle = (sideA + sideB) + sideC;

if(triangle == 180){
    triangle = true;
    console.log(triangle);
} else if (triangle != 180 ) {
    triangle = false;
    console.log(triangle);
} if (sideA < 0 || sideB < 0 || sideC < 0) {
    console.log('Erro um dos lados do triangulo é negativo');
}

File: Modulo-Ciencias-Da-Computação/bloco_32/day_One/Exercises/triangle.py
def triangle(side_a, side_b, side_c):
    is_triangle = (
        side_a + side_b > side_c
        and side_b + side_c > side_a
        and side_a + side_c > side_b
    )
    if not is_triangle:
        return "Não é um triangulo válido"

    elif side_a == side_b == side_c:
        return "Triangulo Equilátero"

    elif side_a == side_b or side_b == side_c or side_a == side_c:
        return "Triangulo Isósceles"

    else:
        return "Triangulo Escaleno"


print(triangle(1, 1, 11))
